src/components/Profile/ProfileComponent.jsx

```diff
@@ -116,13 +124,29 @@ const ProfileComponent = ({ contactProfile }) => {
             setInputValue={setNickname}
             edit={edit}
           />
+          <ProfileInputField inputName="WebId" inputValue={webId} />
```

Comment:
The `WebId` does not display in full. The user has to manually scroll to the right in order to see the entire thing. Not sure if we care about that. Or if we're just going to display the username.

<img width="550" alt="2024-02-19 (2)" src="https://github.com/codeforpdx/PASS/assets/83156697/a32f34f4-9b9d-4021-8281-5794d0432c02">

There are some other things that come to mind, such as when clicking on the edit button it stretches out the box but none of the fields extend along with it.

The content copy icon could maybe be done in line with the WebId itself? MUI should has an option for that. The `AddContactModal` already does, although that is not currently functional.

It does this with InputProps

```
InputProps={{
  endAdornment: (
    <InputAdornment position="end">
      <IconButton aria-label="Copy WebId" edge="end">
        <ContentCopyIcon />
      </IconButton>
    </InputAdornment>
  )
}}
```

But I'm wondering if all this could just be left for another PR. I would like UX to go over the whole profile page, which would not be necessary to get this one done.

Reply:
Added a copy to clipboard button next to the web ID. 

The web ID can often get quite long, so we will need to cut it off somewhere. Cutting it off at the same point as the other fields seems reasonable.

Changing the webid when you click "edit" may make the user think the webID is editable, which it is not.

I'd also like to get feedback from UX about what they think about what they think of the app, but I haven't heard anything from them in a while. In the mean time, we can come up with our own designs.